Sources/NodeAPIMacros/NodeModuleMacro.swift

```diff
@@ -1,3 +1,4 @@
+import Foundation
```

Comment:
I'd like to avoid importing Foundation in the macros to keep them lightweight. Thankfully it doesn't seem strictly necessary here, should be relatively trivial to do the backslash-escaping without it.